aiida/manage/manager.py

```diff
@@ -111,6 +112,10 @@ def _load_backend(self, schema_check=True):
             from aiida.orm.implementation.sqlalchemy.backend import SqlaBackend
             self._backend = SqlaBackend()
 
+        # Reconfigure the logging with `with_orm=True` to make sure that profile specific logging configuration options
+        # are taken into account and the `DbLogHandler` is configured.
+        configure_logging(with_orm=True)
```

Comment:
Yes it will. The idea is that in principle we always want the logging to be configured as soon as a profile is loaded but we do not want to configure the `DbLogHandler` because that requires loading the database. This needs to be done then when the backend is loaded. I don't see any way around this. Unless we remove the requirement that the logging is configured as soon as a profile is loaded, which seems undesirable, but might be ok, I am not sure

Reply:
I see. Perhaps just adding a bit of documentation might help for the moment, i.e.
 * extending the comment where `configure_logging` is used without `with_orm`
 * documenting the `with_orm` parameter inside `configure_logging` (seems to be missing).

Reply:
will do